pkg/builder/controller.go

```diff
@@ -125,6 +126,14 @@ func (blder *Builder) WithEventFilter(p predicate.Predicate) *Builder {
 	return blder
 }
 
+// WithControllerOptions overrides the controller options use in doController. Defaults to empty.
+//
+// Deprecated: Use ControllerManagedBy(Manager) and this isn't needed.
+func (blder *Builder) WithControllerOptions(options controller.Options) *Builder {
```

Comment:
`WithOptions`?

Reply:
Done

Reply:
Actually wasnt sure about WithOptions and WithControllerOptions since Manager too has options and it could be overriden in the future maybe?